SU-SWS/saa_victoria_subtheme
# Adapt 000  ol styles

# READY FOR REVIEW

# Summary
- adds fix so all types of ordered lists work

# Review By (Date)
- 7/29

# Urgency
- soon - waiting for it

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. check with this code and make sure all 5 types work


<h2>List with type=A</h2>

<ol class="list-A" type="A">
	<li>Item</li>
	<li>here</li>
	<li>more</li>
</ol>

<h2>List with type=a</h2>

<ol class="list-a" type="a">
	<li>Item</li>
	<li>here</li>
	<li>more</li>
</ol>

<h2>List with type=I</h2>

<ol class="list-I">
	<li>Item</li>
	<li>here</li>
	<li>more</li>
</ol>

<h2>List with type=i</h2>

<ol class="list-i">
	<li>Item</li>
	<li>here</li>
	<li>more</li>
</ol>

<h2>List with number plain</h2>

<ol >
	<li>Item</li>
	<li>here</li>
	<li>more</li>
</ol>


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?
N/A

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
